src/server.js

```diff
@@ -2,10 +2,10 @@ const express = require("express");
 const cors = require("cors");
 const helmet = require("helmet");
 const Knex = require("knex");
-const port = process.env.PORT || 3005;
 const bodyParser = require("body-parser");
 const { Model } = require("objection");
 const { dbConfig } = require("./config");
+const port = process.env.APP_PORT || 3005;
```

Comment:
the value for `process.env.APP_PORT` is loaded from the `.env` file in the logic required just before this line; `const { dbConfig } = require("./config");` Before this change, configuring a value of anything other than `3005` didn't have any efffect.